prometheus-exporters/install-exporter-amd64.sh

```diff
@@ -816,6 +865,21 @@ EOF
     ]
   }
 ]
+EOF
+  fi
+
+  if [ "$EXPORTER" == "rds" ]; then
+    cat << EOF > /etc/opsverse/targets/${EXPORTER}-exporter.json
+[
+  {
+    "labels": {
+      "job": "integrations/rds-exporter"
+    },
+    "targets": [
+      "localhost:9042"
```

Comment:
@aditya-opsverse, I thought that the scrape path was something other than the default `metrics` ? If so, we may have to adjust this `json`.

Reply:
I didn't quite get this Arul.